extensions/2.0/Vendor/EXT_mesh_features/schema/featureId.schema.json

```diff
@@ -20,6 +20,11 @@
             "minimum": 0,
             "description": "A value that indicates that no feature is associated with this vertex or texel."
         },
+        "label": {
+            "type": "string",
+            "pattern": "^[a-zA-Z]\\w*$",
```

Comment:
are we disallowing `_` at the start because it could get confusing when translating to a shader value? e.g. things might become `featureId__label` with 2 underscores in such cases?

If not, I'd change the part in the square brackets to `[_a-zA-Z]`

Reply:
No, that was just an oversight. I'll push a fix.